app.js

```diff
@@ -378,16 +417,16 @@ fastify.after(() => {
     '/explorer/v1/groups/:projectId/clusters/connectionInfo',
     (request, reply) => {
       reply.send(
-        mongoURIs.map(({ uri, id }) => ({
+        mongoURIs.map(({ uri, id, clientConnectionString }) => ({
```

Comment:
It would be better to resolve SRV at `connectionInfo` endpoint. If SRV resolution changes we don't need to restart the server